### PR TITLE
[13.0] [ADD] stock_picking_product_barcode_report_secondary_unit: Extend stock_picking_product_barcode_report for take care about secondary units

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT25884

This module extend the functionality of stock_picking_product_barcode_report for set by default the maximum number of labels depending of the secondary unit with minimal factor. On next image you can see an example of the wizard.

image

please @sergio-teruel @carlosdauden review this.

It depends on https://github.com/OCA/stock-logistics-barcode/pull/300

### CODE REVIEW

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Can you increase the module coverage

**Comment by CarlosRoca13** [comment]
@sergio-teruel Done!

**Review by sergio-teruel — APPROVED** [review]

File: stock_picking_product_barcode_report_secondary_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    :target: https://translation.odoo-community.org/projects/stock-logistics-barcode-13-0/stock-logistics-barcode-13-0-stock_picking_product_barcode_report_secondary_unit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/150/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extend the functionality of stock_picking_product_barcode_report to set by
+default the maximum qty of labels depending of the secondary unit.
```
Comment (pedrobaeza) [inline stock_picking_product_barcode_report_secondary_unit/README.rst]:
```suggestion
This module extends the functionality of stock_picking_product_barcode_report to set by
default the maximum qty of labels depending on the secondary unit.
```

File: stock_picking_product_barcode_report_secondary_unit/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+#. Active on 'Settings > Inventory' the option Units of Measure
```
Comment (pedrobaeza) [inline stock_picking_product_barcode_report_secondary_unit/README.rst]:
```suggestion
#. Go to *Inventory > Settings*.
#. Activate the option "Units of Measure".
```

**Comment by CarlosRoca13** [comment]
@pedrobaeza changes done!

**Pull request merged**